lib/ruby_lsp/requests/code_action_resolve.rb

```diff
@@ -206,8 +267,6 @@ def refactor_method
         )
       end
 
-      private
```

Comment:
intentional?

Reply:
Yeah, we had added a new method that should be private as public, so I moved it.